bridge/client/app/_components/ktb-no-service-info/ktb-no-service-info.component.ts

```diff
@@ -1,26 +1,31 @@
-import { Component, OnDestroy, OnInit } from '@angular/core';
-import { takeUntil } from 'rxjs/operators';
-import { DataService } from '../../_services/data.service';
+import { Component, OnDestroy } from '@angular/core';
+import { filter, map, takeUntil } from 'rxjs/operators';
 import { Subject } from 'rxjs';
+import { ActivatedRoute } from '@angular/router';
+import { Location } from '@angular/common';
 
 @Component({
   selector: 'ktb-no-service-info',
   templateUrl: './ktb-no-service-info.component.html',
   styleUrls: [],
 })
-export class KtbNoServiceInfoComponent implements OnInit, OnDestroy {
+export class KtbNoServiceInfoComponent implements OnDestroy {
   private unsubscribe$: Subject<void> = new Subject();
-  public isQualityGatesOnly = false;
+  public routerLink = '';
```

Comment:
`routerLink` sounds very generic, maybe better rename it to `servicesSettingsLink` or something similar.